### PR TITLE
[DDID-581]: Add a readiness/liveness probe to clamav.

### PR DESCRIPTION
**Pull request by steveworley**
- Adds a script that can be run to determine pod health.

### CODE REVIEW

File: bay/images/clamav/readiness.sh [hunk 1]
```diff
@@ -0,0 +1,17 @@
+#! /bin/sh
+
+set -e
+
+if freshclam | grep -q 'bytecode.cvd database is up to date'; then
+  echo 'freshclam running successfully'
+  if clamdscan eicar.com | grep -q 'Infected files: 1'; then
```
Comment (nicksantamaria) [inline bay/images/clamav/readiness.sh]: Is this actually testing that the service is up and accepting connections though?

File: bay/images/Dockerfile.clamav [hunk 2]
```diff
@@ -8,24 +8,33 @@ RUN apk add --no-cache -u --repository http://dl-cdn.alpinelinux.org/alpine/edge
...
 RUN curl -sLo /var/lib/clamav/main.cvd http://database.clamav.net/main.cvd && \
   curl -sLo /var/lib/clamav/daily.cvd http://database.clamav.net/daily.cvd && \
   curl -sLo /var/lib/clamav/bytecode.cvd http://database.clamav.net/bytecode.cvd && \
   chown clamav:clamav /var/lib/clamav/*.cvd && \
-  mkdir /run/clamav
+  mkdir /run/clamav && \
+  chown -R clamav:clamav /run/clamav
+
+COPY --chown=clamav:clamav clamav/eicar.com /
+COPY --chown=clamav:clamav clamav/readiness.sh /
```
Comment (nicksantamaria) [inline bay/images/Dockerfile.clamav]: Can we configure a pod readiness script for Lagoon?

**Comment by nicksantamaria** [comment]
I'm researching whether we can set up a readiness probe on the pods for this app.

File: bay/images/Dockerfile.clamav (hunk 2 above)
Comment (nicksantamaria) [reply bay/images/Dockerfile.clamav]: Just checked with amazee and while this is technically possible with Openshift, its not currently supported in lagoon on native kubernetes.

**Comment by steveworley** [comment]
Moved to #43

**Pull request closed without merging**